docs/src/auth.md

```diff
@@ -258,7 +258,7 @@ await context.addInitScript(storage => {
 
 ```java
 // Get session storage and store as env variable
-String sessionStorage = (String) page.evaluate("() => JSON.stringify(sessionStorage");
+String sessionStorage = (String) page.evaluate("() => JSON.stringify(sessionStorage)");
```

Comment:
we could also simplify it to `page.evaluate("JSON.stringify(sessionStorage)")`, but it's up to you

Reply:
Thanks for the proposal.
Added another commit to the PR.

Reply:
Awesome, and congrats with your first Pull Request! 💯